pkg/install/install.go

```diff
@@ -35,6 +37,14 @@ func (obj *installer) InstallBlocking(ctx context.Context, pkg *v1alpha1.Package
 	if err != nil {
 		return nil, err
 	}
+	if obj.dryRun {
+		time.Sleep(6 * time.Second)
```

Comment:
I think waiting is not necessary for this feature, since `dry-run` is intended to be mostly used in automation.